src/mococrw/ecies.h

```diff
@@ -142,6 +142,16 @@ class ECIESEncryptionCtx {
     std::unique_ptr<Impl> _impl;
 };
 
+/**
+ * @brief Get encoded key length in bytes
+ *
+ * Depending on the key length and the elliptic curve point conversion form the key will take different amount
+ * of bytes when encoded. This function can be used to conveniently deduct the encoded length while deserializing.
+ * @param privKey Private key
```

Comment:
I guess we should document that this private key is really just used to figure out what key size is used in this instance of ECIES. Can you add a comment explaining that?

While you're at it, the rest of the code in this file does not add a space between function name and parameter list, so I'd prefer if we used the same style in this change.

Reply:
Done